### PR TITLE
Issue801 outdoor airflow setpoint

### PR DESCRIPTION
**Pull request by JayHuLBL**
Blocks to calculate outdoor airflow rate set point, single zone and multilane system.

### CODE REVIEW

**Comment by JayHuLBL** [comment]
Following updates have been done:
- [x] add comment to all instances
- [x] reorder contents so to have following orders: parameters, inputs, outputs, instances.
- [x] declare inputs/outputs
- [x] use shorter names for models in validation package
- [x] add package information documentation
- [x] add ASHRAE G36 reference

**Comment by JayHuLBL** [comment]
@mwetter The model would be ready for review if Travis test is successful.

**Comment by JayHuLBL** [comment]
@mwetter The pull request is ready for review.

File: Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/OutdoorAirFlowSetpoint_MultiZone.mo [hunk 1]
```diff
@@ -0,0 +1,656 @@
...
+    annotation (Placement(transformation(extent={{-168,250},{-148,270}})));
+  CDL.Continuous.Sources.Constant maxSysPriFlow(k=maxSysPriFlo)
+    "Highest expected system primary airflow"
+    annotation (Placement(transformation(extent={{20,140},{40,160}})));
+  CDL.Continuous.Sources.Constant zerPriAir[numOfZon](k=fill(0.1, numOfZon))
+    "Near zero primary airflow so to avoid divide-by-zero issue in later process when supply fan is off"
+    annotation (Placement(transformation(extent={{-120,-200},{-100,-180}})));
+
+equation
+  for i in 1:numOfZon loop
```
Comment (JayHuLBL) [inline Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/OutdoorAirFlowSetpoint_MultiZone.mo]: @mwetter I have updated the connections for blocks with same dimensions. But there are also connections that connect one boolean output to a vectorized input, like the  `connect(not1.y, swi3[i].u2)` which sends supply fan status signal for calculations in each zone level. 
There is block like `Modelica.Blocks.Routing.BooleanReplicator` can be used for the case. Should it be included into CDL library?

**Comment by mwetter** [comment]
@JayHuLBL 
Yes, we will need a package
CDL.Routing
with BooleanReplicator, RealReplicator etc. 
Can you please implement it

**Comment by mwetter** [comment]
This is ready to merge when all tests pass.

**Pull request merged**